test/core/integration/concrete/lockup-base/withdraw-multiple/withdrawMultiple.tree

```diff
@@ -6,7 +6,7 @@ WithdrawMultiple_Integration_Concrete_Test
    │  └── it should revert
    └── when equal arrays length
       ├── when zero array length
-      │  └── it should do nothing
+      │  └── it should charge the fee
```

Comment:
this is not being tested in the test contract so no need to mention that it should charge the fee.

Reply:
That is incorrect, the function that runs the balanceTest modifier is being is used

Reply:
Oh. Usually we test assertions at the end of the logic. So thats where I looked for it.